### PR TITLE
Update monitors.py

### PR DESCRIPTION
**Pull request by alanyee**
Refactor contrib

### CODE REVIEW

File: tensorflow/contrib/learn/python/learn/monitors.py [hunk 1]
```diff
@@ -54,7 +51,7 @@
 from tensorflow.python.training import saver as saver_lib
 from tensorflow.python.training import session_run_hook
 from tensorflow.python.training import training_util
-from tensorflow.python.util import deprecation
+from tensorflow.python.util.deprecation import deprecated
```
Comment (martinwicke) [inline tensorflow/contrib/learn/python/learn/monitors.py]: We generally prefer to import the whole file.

File: tensorflow/contrib/learn/python/learn/monitors.py [hunk 2]
```diff
@@ -42,9 +42,6 @@
 import numpy as np
 import six
 
-from tensorflow.contrib.framework import deprecated
-from tensorflow.contrib.framework.python.ops import variables as contrib_variables
-from tensorflow.contrib.learn.python.learn.summary_writer_cache import SummaryWriterCache
```
Comment (martinwicke) [inline tensorflow/contrib/learn/python/learn/monitors.py]: All those are not needed? Good find!

**Review by martinwicke — CHANGES_REQUESTED** [review]

File: tensorflow/contrib/learn/python/learn/monitors.py (hunk 2 above)
Comment (alanyee) [reply tensorflow/contrib/learn/python/learn/monitors.py]: Ah I actually update this code some time ago, and realized that these import statements were no longer needed.

**Comment by yifeif** [comment]
@tensorflow-jenkins test this please

**Comment by alanyee** [comment]
@martinwicke does this look good?

**Review by martinwicke — APPROVED** [review]

**Pull request merged**